modules/core/src/passes/pick-layers-pass.js

```diff
@@ -19,7 +19,8 @@ export default class PickLayersPass extends LayersPass {
     onViewportActive,
     pickingFBO,
     deviceRect: {x, y, width, height},
-    redrawReason = ''
+    redrawReason = '',
+    effectProps = {lightSources: {}}
```

Comment:
Why is the default value needed? The picking pass should not be aware of specific effects.

Reply:
didn't we discuss to use the empty lightSources to disable lighting? if we don't fill it here, it can be in deck-picker or base LayersPass, where do you prefer?